extensions/amp-ad-network-fake-impl/0.1/external-reorder-head-transformer.js

```diff
@@ -0,0 +1,282 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {endsWith, startsWith} from '../../../src/string';
+import {urls} from '../../../src/config';
+
+export class ExternalReorderHeadTransformer {
+  /** constructor */
+  constructor() {
+    /** @private {!Object} */
+    this.headComponents_ = {
+      metaOther: [],
```

Comment:
If you put types here you can use includes. Switching to indexOf is fine too. Whatever you prefer.